Python/Tests/Utilities.Python/PythonToolsTestUtilities.cs

```diff
@@ -104,9 +106,29 @@ public static class PythonToolsTestUtilities {
             return serviceProvider;
         }
 
-        private static PythonEditorServices CreatePythonEditorServices(IServiceContainer site) {
+        class LazyComponentGetter<T> : Lazy<T> {
+            public LazyComponentGetter(MockComponentModel model) : base(() => (T)model.GetService(typeof(T))) { }
+        }
+
+        private static PythonEditorServices CreatePythonEditorServices(IServiceContainer site, MockComponentModel model) {
             var services = new PythonEditorServices(site);
-            services.ComponentModel.DefaultCompositionService.SatisfyImportsOnce(services);
+
+            //services.ComponentModel.DefaultCompositionService.SatisfyImportsOnce(services);
```

Comment:
>services [](start = 14, length = 8)

delete?

Reply:
I kind of like that it shows what we're emulating... maybe I'll add that to the comment too.